src/core/public/chrome/ui/header/header.tsx

```diff
@@ -72,162 +73,72 @@ export interface HeaderProps {
   basePath: HttpStart['basePath'];
   isLocked$: Rx.Observable<boolean>;
   navType$: Rx.Observable<NavType>;
+  loadingCount$: ReturnType<HttpStart['getLoadingCount$']>;
```

Comment:
Why's that? I thought it would be nice to have a single source of truth on the type.

Reply:
IDK, either way, TS compilation would fail if `HttpStart['getLoadingCount$']` type changes (just not in the same file), and I find it way more readable. But it's a personal option, you can keep it like that.

(The same way I would replace `HttpStart['basePath']` by it's `IBasePath` actual type)

Reply:
👍 I'll keep it like this for now then. If other people have opinions on this, would be nice to come to a consensus on what we should prefer in the Kibana repo.